zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js

```diff
@@ -400,4 +434,28 @@ export function PrevArrow(props) {
   );
 }
 
+export function EnlargedImgArrow (props) {
+  const { className, style, onClick } = props;
+  return (
+    <div
+      className={className}
+      style={{
```

Comment:
avoid inline styles, instead use use the classes. if some style needs to be set dynamically use css properties